tests/SvgTest.php

```
@@ -41,6 +41,28 @@
 #[Group('svg')]
 final class SvgTest extends TestCase
 {
+    public function testIconPathReturnsAbsolutePathFromCollectionReference(): void
+    {
+        self::assertSame(
+            dirname(__DIR__) . '/assets/icons/Bootstrap5/globe.svg',
+            Svg::iconPath('Bootstrap5:globe'),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Svg\Svg' in method 'testIconPathReturnsAbsolutePathFromCollectionReference'. (undefined)

(StaticAccess)